radis/gpu/gpu.py

```diff
@@ -0,0 +1,398 @@
+    na : numpy.ndarray[np.float32]
+        Temperature dependency of Lorentzian widths
+    S0 : numpy.ndarray[np.float32]
+        Line intensity scaling factors.
+    El : numpy.ndarray[np.float32]
+        Lower level energy levels.
+    gamma_arr : numpy.ndarray[np.float32]
+        (m,n) shaped array with Lorentzian width parameters, with n the number of lines in the database
+        and m the number of collision partners included. This is usually at least two,
+        with the first (m=0) always self broadening and the last (m=-1) always air broadening.
+    iso : numpy.ndarray[np.uint8]
+        Index of isotopologue.
+    Mm_arr : numpy.ndarray
+        Molecular masses for all isotopologues in the database (Mm_arr[0] is always 0).
+    Q_intp_list : list
+        List of Q branch interpolators.
+    verbose : bool, optional
+        Print verbosity level. Default is 0.
+    backend :  ``'gpu-cuda'``, ``'cpu-cuda'``, optional
+        Which backend to use; currently only CUDA backends (Nvidia) are supported. ``'cpu-cuda'`` runs the kernel on CPU. Default is ``'gpu-cuda'``.
+
+    Returns
+    -------
+    init_h : radis.gpu.structs.initData_t
+        structue with parameters used for GPU computation that are constant
+        during iterations.
+    """
+
+    global gpu_mod
+    if gpu_mod is not None:
+        warn("Only a single GPU context allowed; please call gpu_exit() first.")
+        return
+
+    ## First a GPU context is created, then the .ptx file is read
+    ## and made available as the GPUModule object gpu_mod
+    ## If this fails, None is returned and calculations are
+    ## defaulted to CPU emulation
+
+    if backend == "cpu-cuda":
+        from radis.gpu.cuda.emulate import CuContext as GPUContext
+
+        ctx = GPUContext.Open(verbose=verbose)
+        import radis.gpu.cuda.emulate as backend_module
+
+    else:
+        # Try to load GPU
+        from radis.gpu.cuda.driver import CuContext as GPUContext
+
+        ctx = GPUContext.Open(verbose=verbose)  # Set verbose to >=2 for comments
+        if ctx is None:
+            warn(
+                NoGPUWarning(
+                    "Failed to load CUDA context, this happened either because"
+                    + "CUDA is not installed properly, or you have no NVIDIA GPU. "
+                    + "Continuing with emulated GPU on CPU..."
+                    + "This means *NO* GPU acceleration!"
+                )
+            )
+
+            # failed to init CUDA context, continue with CPU:
+            from radis.gpu.cuda.emulate import CuContext as GPUContext
+
+            ctx = GPUContext.Open(verbose=verbose)
+            import radis.gpu.cuda.emulate as backend_module
+
+        else:
+            # successfully initialized CUDA context, continue with GPU:
+            import radis.gpu.cuda.driver as backend_module
+
+    GPUContext, GPUModule, GPUArray, GPUFFT, GPUTimer = backend_module.getClasses()
+
+    if verbose == 1:
```

Comment:
```suggestion
    if verbose:
```
I guess you want the print also for verbose = 2, 3 etc.